Sources/BraintreePayPal/RecurringBillingMetadata/BTPayPalRecurringBillingDetails.swift

```diff
@@ -36,8 +38,9 @@ public struct BTPayPalRecurringBillingDetails {
         productQuantity: Int?,
         oneTimeFeeAmount: String?,
         shippingAmount: String?,
-        productAmount: String?,
-        taxAmount: String?
+        productPrice: String?,
+        taxAmount: String?,
+        totalAmount: String
```

Comment:
Total amount is required here and optional above. Should these be the same?

Reply:
Yes, it should be a required property.

Reply:
Cool, can we move it to the top of the init with the other required params? Along with moving up the docstring.

Reply:
✅